valhalla/baldr/traffictile.h

```diff
@@ -137,46 +137,55 @@ struct TrafficSpeed {
         congestion3{c3}, has_incidents{incidents}, spare{0} {
   }
 
-  json::MapPtr json() const volatile {
-    auto live_speed = json::map({});
+  void json(rapidjson::writer_wrapper_t& writer) const volatile {
```

Comment:
What do you think about completely moving this function from `valhalla/baldr/traffictile.h` to `src/tyr/locate_serializer.cc`?
This will allow to remove `#include <valhalla/baldr/rapidjson_utils.h>` from here, which would remove it from many sources because of how often `valhalla/baldr/traffictile.h` is included via `graphtile.h`.